### DISCUSSION TITLE
怎么提供测试报告，机型、表现、日志之类的？

### DISCUSSION THREAD

**Discussion by ceasarXuu** [Geocld/XStreaming#4] [General]
我的两个机器测试连接成功率不高，目前只有一台机器成功连接过2次，以及D8手柄在地平线5有按键映射错误的问题，如何提交测试问题

**Comment by Geocld** [comment]
@ceasarXuu 可以直接到[issues](https://github.com/Geocld/XStreaming/issues)提交问题，尽量附上截图、机型、操作系统等环境信息，谢谢。
